chat.delta.desktop.yml

```diff
@@ -21,6 +21,9 @@ finish-args:
   - --filesystem=xdg-videos
   - --share=network
   - --talk-name=org.freedesktop.Notifications
```

Comment:
```suggestion
  - --talk-name=org.freedesktop.Notifications # needed for dbus notification, as the notification portal crashed because of a bug in electron https://github.com/electron/electron/issues/40461
```